### ISSUE TITLE
<aside> more appropriate than <figure> for wrapping Pull Quote

### ISSUE THREAD

**Issue by m-e-h** [WordPress/gutenberg#14606]
**Describe the bug**
Currently the PullQuote block outputs this HTML:
```
<figure class="wp-block-pullquote">
  <blockquote>
    <p>Hello Pull Quote</p>
  </blockquote>
</figure>
```

The HTML5 specifications suggest it should look something like this instead:
```
<aside class="wp-block-pullquote">
  <blockquote>
    <p>Hello Pull Quote</p>
  </blockquote>
</aside>
```
[The W3C docs for the figure element have this to say:](https://www.w3.org/TR/html50/grouping-content.html#the-figure-element)

> For content that is only tangentially related, or that serves a separate purpose than the surrounding flow, the aside element should be used (and can itself wrap a figure). 

> For example, a pull quote that repeats content from an article would be more appropriate in an aside than in a figure, because it isn't part of the content, it's a repetition of the content for the purposes of enticing readers or highlighting key topics.

**Comment by youknowriad** [comment]
Looks like this was already discussed and abandoned here https://github.com/WordPress/gutenberg/pull/9599 

cc @afercia @jasmussen

**Comment by afercia** [comment]
> The HTML5 specifications suggest it should look something like this instead:

Not sure it does in the definition of the `blockquote` element:
https://www.w3.org/TR/html52/grouping-content.html#the-blockquote-element

Instead, there's one example of blockquote within an aside in the `aside` element definition: https://www.w3.org/TR/html52/sections.html#the-aside-element
That's a very specific case: tweets are certainly "tangentially related" to the main content. Instead, inline quotes are part of the content.

@m-e-h the point you raised is valid, but only in one specific case. The bit you quoted from the figure element documentation makes it clear:
> For content **that is only tangentially related**, **or that serves a separate purpose** than the surrounding flow, the aside element should be used (and can itself wrap a figure). For example, a pull quote that **repeats** content from an article would be more appropriate in an aside than in a figure, **because it isn’t part of the content** ...

Theoretically, there should be a way to distinguish when a quote is or isn't part of the main content.

Regardless, there's an important consequence to consider. The `<aside>` element has an [intrinsic role of complementary](https://www.w3.org/TR/html-aria/#aside) thus it's actually an [ARIA landmark](https://www.w3.org/TR/wai-aria-1.1/#landmark_roles). Landmarks are meant to indicate [main regions of a page](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html) to facilitate navigation for assistive technologies users. It's essential to keep landmarks to a reasonable, predictable, amount so that users can can actually use them.

See also https://github.com/WordPress/gutenberg/pull/9599#issuecomment-418283145

**Comment by m-e-h** [comment]
Sorry for the duplicate. I should've searched a little better. I came across that W3C recommendation while researching something unrelated and wanted to open an issue here before I forgot. 

I do think that `aside` is the correct way to do Pull Quotes. In reading through all the related W3 recommendations linked here, I only see "Pull Quotes" mentioned once and an `aside` is what's suggested.

Wouldn't something that is
> for the purposes of enticing readers or highlighting key topics.

be considered a [main region of a page](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html)?

You do make an excellent point, though, @afercia and it leads me to think `aside` probably isn't such a good idea here.
> It's essential to keep landmarks to a reasonable, predictable, amount

When choosing between a Quote and a Pull Quote, I sometimes just select the first one I see or whichever is styled the way I want. I'm assuming the typical user might do the same thing. 
This assumed usage makes Pull Quotes anything but "reasonable" or "predictable".

Probably best to stick to what we currently have. I'm thinking we should close this issue since it was already discussed in https://github.com/WordPress/gutenberg/pull/9599 . Any reason to leave it open?

**Comment by afercia** [comment]
> Any reason to leave it open?

Kept open to be fair and and allow you to reply :) Closing now, if no objections.

**Comment by AlchemyUnited** [comment]
I realize this is closed but...

It would seem to me <figure> is less appropriate than <aside>. That said, if as @afercia mentions aside is a ARIA landmark then I would think it could be <aside aria-hidden=true>. For all intents and purposes, a blockquote is a decorative image, and decorative images are supposed to be tagged so AT ignores them. That is, a blockquote is more for visual aesthetics than actually adding content value, as the content has already been - or will be - presented on the page in a paragraph or perhaps a list. 

In short, I would think it's rare that AT needs to be aware of a blockquote; in the same way AT doesn't need to be aware of decorative images.
